FirebaseVertexAI/Sources/VertexAI.swift

```diff
@@ -0,0 +1,112 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+import FirebaseAppCheckInterop
+import FirebaseCore
+
+// Exports the GoogleGenerativeAI module to users of the SDK.
+@_exported import GoogleGenerativeAI
+
+// Avoids exposing internal FirebaseCore APIs to Swift users.
+@_implementationOnly import FirebaseCoreExtension
+
+@available(iOS 15.0, macOS 11.0, macCatalyst 15.0, *)
+@objc(FIRVertexAI)
+open class VertexAI: NSObject {
+  // MARK: - Public APIs
+
+  /// Returns an instance of `GoogleGenerativeAI.GenerativeModel` that uses the Vertex AI API.
+  ///
+  /// This instance is configured with the default `FirebaseApp`.
+  public static func generativeModel(modelName: String, location: String) -> GoogleGenerativeAI
+    .GenerativeModel {
+    return generativeModel(app: FirebaseApp.app()!, modelName: modelName, location: location)
```

Comment:
Force unwrap makes me feel uneasy...

Reply:
Added a `fatalError` instead of a force-unwrap. The following error message will also be logged in this scenario: https://github.com/firebase/firebase-ios-sdk/blob/86ae5deb1229c66e0c7d94571fa0685cdcd7b5cc/FirebaseCore/Sources/FIRApp.m#L238-L245